src/main/java/org/snomed/snowstorm/core/data/domain/jobs/ExportStatus.java

```diff
@@ -0,0 +1,9 @@
+package org.snomed.snowstorm.core.data.domain.jobs;
+
+public enum ExportStatus {
+    PENDING,
+    IN_PROGRESS,
+    READY_FOR_DOWNLOAD,
```

Comment:
@dmcgihtsdo Can we have PENDING, RUNNING, COMPLETED,FAILED, DOWNLOADED please?

Reply:
Thanks @CoderMChu, I have made that change.